quickstarts/bagisto/config.yml

```diff
@@ -0,0 +1,49 @@
+id: 9d758ec4-a261-06ef-be55-2e1f9cc55f2e
```

Comment:
This ID will be generated after it is merged into the system. Let's remove this field for now.
```suggestion
```